backend/functions/src/matching/routes.ts

```diff
@@ -28,8 +39,12 @@ router.post('/make', (req, res) => {
 router.get('/:courseId', (req, res) => {
   const courseId = req.params.courseId
   getGroups(courseId)
-    .then((data) => res.status(200).json(data))
+    .then((data) => {
+      logger.info('Successfully attempted to get course info for ' + courseId)
+      res.status(200).json(data)
+    })
```

Comment:
This hasn't logged during my testing, is this route actually being used?

Reply:
I don't believe so either. Just made it just in case we aren't able to load in course data at any point, I guess just being better safe than sorry? 

Honestly, I keep forgetting string template literals exist, thank you. Makes it much cleaner and easier to write. Fixed it literals. 

As for passing objects into logging statements, I didn't really know which objects to pass in, so I didn't pass much in except for errors so far.